### PR TITLE
change back conf.env.properties

### PR DESCRIPTION
**Pull request by tungrd**

### CODE REVIEW

**Review by ake2l — COMMENTED** [review]
maybe we should write this file dynamically to be not affected by local changes

**Comment by tungrd** [comment]
> maybe we should write this file dynamically to be not affected by local changes , would you like to give it a try ?

Ok, I will try it.

**Comment by tungrd** [comment]
I made an update to fix the fail test (E2ET_postgres), and add function to handle postgresql composite data type. I have to remove Geometry type because it will make error geometry does not exist. That problem is cause by test environment postgresql database do not have extension data.

File: src/main/assembly/dist-assembly.xml [hunk 1]
```diff
@@ -198,6 +198,8 @@
                 <include>org.apache.logging.log4j:*:jar</include>
                 <include>org.slf4j:*:jar</include>
                 <include>com.github.javafaker:*:jar</include>
+                <include>net.datafaker:*:jar</include>
```
Comment (ake2l) [inline src/main/assembly/dist-assembly.xml:201]: Great you took care on this ... please also add these dependencies to our Enterprise Edition

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -808,10 +809,20 @@ jobs:
           dbSchema=public
           idStrategy=increment
           idParam=1010" > $ARTIFACT_FULLNAME/demo/shop/postgres/shop.postgres.properties
+      - name: set env file for postgres connection
```
Comment (ake2l) [inline .github/workflows/ci.yml:812]: you should also change to environemt file in quickcheck.yml

**Review by ake2l — COMMENTED** [review]
anything else looks good to me

**Comment by ake2l** [comment]
pipeline run into errors in e2et postgres .... so i am going to merge it now

**Pull request merged**